appinventor/components/src/com/google/appinventor/components/runtime/ChartDataBase.java

```diff
@@ -122,18 +61,6 @@ protected ChartDataBase(Chart chartContainer) {
     threadRunner = Executors.newSingleThreadExecutor();
```

Comment:
Since the threadRunner is initialized in `DataCollection()` we shouldn't create another one here.